ElementX/Sources/Application/AppSettings.swift

```diff
@@ -164,6 +165,9 @@ final class AppSettings: ObservableObject {
     @UserSetting(key: UserDefaultsKeys.startChatFlowEnabled.rawValue, defaultValue: false, persistIn: store)
     var startChatFlowEnabled
     
+    @UserSetting(key: UserDefaultsKeys.startChatUserSuggestionsEnabled.rawValue, defaultValue: false, persistIn: store)
```

Comment:
I would probably suggest not persisting this one (at least until the feature is brought back into scope for someone to work on).
```suggestion
    @UserSetting(key: UserDefaultsKeys.startChatUserSuggestionsEnabled.rawValue, defaultValue: false, persistIn: nil)
```

Reply:
Oh interesting, I hadn't thought of this. What's your thinking with not persisting it?

Reply:
Ideally we don't want to persist feature flags unless we really need to. The thought being that someone could toggle a load of random flags (that might not play well with each other), get their app in a mess of some unhandled state and end up reporting bugs that don't really exist. I suggested to do the same for `startChatFlowEnabled` as I personally don't see the value in persisting this one either.

Reply:
Ok, got it. Have made it so.

For `startChatFlowEnabled` I think it's ok to persist because we're actively working on this and without persisting you'd have to turn it on again after every local build.

Reply:
Thank you!

You could set the `default` value to `true` using it like a build flag for local builds.

Reply:
Ah, that's true, yes. Well, I'll leave this one alone for now since it's Alfonso's. 😅